freeplane/src/main/java/org/freeplane/main/application/MapViewDockingWindows.java

```diff
@@ -208,9 +209,37 @@ public void internalFrameClosing(InternalFrameEvent e) {
             }
 		};
 
+		addTabsPopupMenu(rootWindow);
 
 	}
 
+	private void addTabsPopupMenu(DockingWindow dockingWindow){
+		dockingWindow.setPopupMenuFactory(new WindowPopupMenuFactory() {
+		   public JPopupMenu createPopupMenu(DockingWindow window) {
+			   if(window instanceof ConnectedToMenuView) {
+				   return null;
+			   }
+			   JPopupMenu menu = new JPopupMenu(window.getTitle());
```

Comment:
Have you added an import for JPopupMenu ?

Reply:
I did, but IntelliJ changed all single imports for one `import javax.swing.*` 
And when I saw that I let that part out of the commit, missing to add the JPopMenu import.

I'm making the changes in my branch and will to do a pull request to learn the correct cooperation flow.

thanks again

Reply:
You do not need to open any new pull request because all changes on your branch are automatically added to this pull request

Reply:
ok. understood.